setup.py

```diff
@@ -112,14 +112,14 @@ def build_extension(self, ext):
         build_args += ['--config', cfg]
 
         # Memcheck (guard if it fails)
-        totalMemory = 4000
+        freeMemory = 4000
         if platform.system() == "Linux":
             try:
-                totalMemory = int(os.popen("free -m").readlines()[1].split()[1])
+                freeMemory = int(os.popen("free -m").readlines()[1].split()[2])
```

Comment:
This in my case returns the used memory:
This is the format of `free -m` on ubuntu.
![image](https://user-images.githubusercontent.com/60790666/109885315-fd2d5300-7c86-11eb-9d71-872f0a944649.png)

Change to : ... `split()[3])`
Didn't check specifically on jetson, but the format of `free -m` should be the same on all linux platforms.

Reply:
Yup. you are right.